re/blah.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/bash
+echo "We are here" $foo $PATH
```

Comment:
Cool, looks like this plus `env.json` would make a good test case! Thanks for testing the various codepaths out.